packages/init/.eslintignore

```diff
@@ -1,3 +1,4 @@
-build
-node_modules
-tests
\ No newline at end of file
+# .gitignore copy
+
+/dist
+/playground/*
```

Comment:
what's playground?

Reply:
It was a live testing environment like `e2e-projects`.

The references have been removed in this commit: https://github.com/prismicio/slice-machine/commit/424fdb1f92a0b803ebaef2389158a78a5ec7d27c

Reply:
That seems like a useful thing to have. Or was it basically the same as the e2e-projecs?